appengine/firebase-tictactoe/README.md

```diff
@@ -0,0 +1,54 @@
+# Tic Tac Toe on Google App Engine Standard using Firebase
+
+This directory contains a project that implements a realtime two-player game of
+Tic Tac Toe on [Google App Engine] Standard, using the [Firebase] database
+for realtime notifications when the board changes.
+
+[Firebase]: https://firebase.google.com
+[Google App Engine]: https://cloud.google.com/appengine
+
+## Prerequisites
+
+* Install [Apache Maven][maven]
```

Comment:
It used to be the case for the old appengine plugin, but I've been using mvn 3.0.5 with our plugin and it hasn't complained.